libs/SalesforceSDK/src/com/salesforce/androidsdk/push/PushNotificationInterface.java

```diff
@@ -37,5 +37,5 @@
  */
 public interface PushNotificationInterface {
 
-	void onPushMessageReceived(RemoteMessage message);
+	void onPushMessageReceived(Map<String, String> data);
```

Comment:
Tweaking the method to support both cases. We can't reconstruct a `RemoteMessage` once we've opened up the package.